cudamapper/tests/Test_CudamapperOverlapperTriggered.cu

```diff
@@ -11,144 +11,12 @@
 #include "gtest/gtest.h"
 
 #include "../src/overlapper_triggered.hpp"
-#include "../src/cudamapper_utils.hpp"
-
-#include "mock_fasta_parser.hpp"
-
-#include <algorithm>
-#include <numeric>
-#include <random>
-
-#include "cudamapper_file_location.hpp"
 
 namespace claragenomics
 {
 namespace cudamapper
 {
 
-TEST(TestCudamapperOverlapperTriggerred, FuseTwoOverlaps)
```

Comment:
Why were these tests removed? They seem unrelated to the changes in the main code. Do we have better tests for this functionality somewhere else?

Reply:
These tests only test `fused_overlaps()` function from `cudamapper_utils.hpp`. This function became unused at some point, probably when @vellamike introduced `Overlapper::post_process_overlaps()` so I also removed the function from `cudamapper_utils.hpp`.
I agree this could have been a separate PR as well.

Reply:
No need for a separate PR - it was just a bit confusing.